control/pages.go

```diff
@@ -55,7 +55,7 @@ func paginateChannels(page uint, perPage uint,
 		}
 	}()
 
-	var channels []Channel
```

Comment:
@mristin 

I changed this as otherwise the resulting JSON would have `chanels=null` for some reason. I thought the two are equivalent but apparently they're not. With the fix in this PR, the JSON `ChannelPage` correctly contains an empty array if there are no channels.